### ISSUE TITLE
Decide on using REST API and/or query API in spec

### ISSUE THREAD

**Issue by jorritspee** [jorritspee/openEHRxNuts#2]
Links: https://specifications.openehr.org/its

**Comment by jorritspee** [comment]
Okay thanks. But you mentioned earlier that not all vendors have implemented the REST-API. Is that something we have to take into account (and how)?

**Comment by joostholslag** [comment]
> Okay thanks. But you mentioned earlier that not all vendors have implemented the REST-API. Is that something we have to take into account (and how)?

Yes. All have implemented rest apis (at least all currently relevant for this project.), but not all use the official openehr specification for those apis. Better has, code24 and Nedap have not (yet). At least partly because they don’t support aql and thus can’t support the query api. 
We should discuss how to handle this.  I think we should focus on the official rest apis, because I expect both companies to implement the rest api at some point. But we should discuss how big the deviations are and wether there are possibilities to make the solution work for non standard openehr rest apis too.
